common/src/main/java/pl/edu/agh/messages/client_communication/ClientMessage.java

```diff
@@ -0,0 +1,9 @@
+package pl.edu.agh.messages.client_communication;
```

Comment:
I prefer not using underscores in packages' names. Consider: client, operations etc.